test_unstructured_ingest/expected-structured-output/box/nested-1/ideas-page.html.json

```diff
@@ -21,6 +21,246 @@
         "i"
       ]
     },
-    "text": "January 2023(Someone fed my essays into GPT to make something that could answer\nquestions based on them, then asked it where good ideas come from.  The\nanswer was ok, but not what I would have said. This is what I would have said.)The way to get new ideas is to notice anomalies: what seems strange,\nor missing, or broken? You can see anomalies in everyday life (much\nof standup comedy is based on this), but the best place to look for\nthem is at the frontiers of knowledge.Knowledge grows fractally.\nFrom a distance its edges look smooth, but when you learn enough\nto get close to one, you'll notice it's full of gaps. These gaps\nwill seem obvious; it will seem inexplicable that no one has tried\nx or wondered about y. In the best case, exploring such gaps yields\nwhole new fractal buds."
+    "text": "January 2023(Someone fed my essays into GPT to make something that could answer"
+  },
+  {
+    "type": "NarrativeText",
+    "element_id": "ff8e75afa0e8e604d1d2fedf2679a352",
+    "metadata": {
+      "data_source": {},
+      "filetype": "text/html",
+      "page_number": 1,
+      "link_urls": [
+        "index.html",
+        "https://twitter.com/stef/status/1617222428727586816"
+      ],
+      "link_texts": [
+        null,
+        null
+      ],
+      "emphasized_text_contents": [
+        "(Someone fed my essays into GPT to make something that could answer\nquestions based on them, then asked it where good ideas come from.  The\nanswer was ok, but not what I would have said. This is what I would have said.)"
+      ],
+      "emphasized_text_tags": [
+        "i"
+      ]
+    },
+    "text": "questions based on them, then asked it where good ideas come from. The"
```

Comment:
These results look suspicious to me. It seems like it breaks when applying `split_by_paragraph` on top of the extracted html text which was not in the intended paragraph format. I also agree that splitting with `<br>` could result in a prettier output :)